docs/docs/working-with-Etsy-in-Gatsby.md

```diff
@@ -0,0 +1,57 @@
+---
+Title: Working with Etsy in Gatsby
+---
+
+## Etsy in Gatsby
+
+This guide will walk you through the steps needed in order to successfully work with Etsy e-commerce sites in Gatsby.
+
+## Prerequisites
```

Comment:
This is in another comment but I'd try to move away from the step-by-step bullets where it makes sense.